fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/LdpContainerRdfContext.java

```diff
@@ -203,9 +203,13 @@ public FedoraResourceTripleFunction(final String insertedContainerProperty,
                 return Iterators.transform(values, new Function<Value, Triple>() {
                     @Override
                     public Triple apply(final Value input) {
-                        final RDFNode membershipResource = new ValueConverter(session(), translator())
-                                .convert(input);
-                        return create(subject(), memberRelation, membershipResource.asNode());
+                        try {
+                            final RDFNode membershipResource = new ValueConverter(resource().getNode().getSession(),
```

Comment:
I suspect the `session()` is not getting set in the first place. Is there danger in updating:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/NodeRdfContext.java#L52
...to pass in to super() `resource.getNode().getSession()`?